webapp/src/components/link_tooltip/tooltip.css

```diff
@@ -86,6 +86,7 @@
 
 /* Labels */
 .github-tooltip .labels {
+    display: flex;
```

Comment:
What effect does this have on the tooltip's styling?

Reply:
`.labels` is not flex element so the properties below doesn't working. So I added it and flex works.

| Before | After |
| --- | --- |
| ![image](https://github.com/mattermost/mattermost-plugin-github/assets/26380261/97bc4a6a-22f0-4540-92fd-fd880d107a2e) | ![image](https://github.com/mattermost/mattermost-plugin-github/assets/26380261/4c66a089-ce1f-4fd5-933b-3c64e9b2a5b7) |
| `.labels = height: 23px; (auto) display: block;`<br>( flex properties not working ) | `.labels = height: 20px; (auto) display: flex;`<br>( flex properties working )
| `.labels > * = height: 20px;` | `.labels > * = height: 20px;` |